src/styles/components/index.js

```diff
@@ -1,12 +1,17 @@
 import styled from "styled-components";
+import * as styleVariables from './variable'
+
+const landingColorPrimary = styleVariables.colorHackRed;
+const landingColorBorder = styleVariables.hackBlack5;
+const landingColorDarkBg = styleVariables.hackBlack5;
```

Comment:
should this be `LandingColorPrimary' etc ?

Reply:
variable name = camel Case, component name = capital by convention i think

Reply:
style variables can be shorten to svar ? stylevar ? etc

Reply:
what about just `style.` or is that used already?

Reply:
style can be confused with styled from gatsy style components

Reply:
`svar` or anything short will work, I'll leave that up to you. I just think `styleVariables` is too long for a prefix

Reply:
I feel like svar is not expressive enough and can be confusing to read too. I'll go with stylevars

Reply:
just fixed it